website/static/js/myProjects.js

```diff
@@ -1810,6 +1786,7 @@ var Information = {
         }
         if (args.selected().length === 1) {
             var item = args.selected()[0].data;
+            var permission = item.attributes.current_user_permissions[item.attributes.current_user_permissions.length-1];
```

Comment:
Because that's the highest permission the users has it's written as a list so it can be evaluated in statements like `if 'read' in permmissions:` the instead of `if permmission == 'read' or permmission == 'write' or permmission == 'admin'`  so the last element in the list will display the highest permission.